ProcessLib/SmallDeformationWithLIE/Common/Utils.cpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#include "Utils.h"
+
+#include <cmath>
+
+#ifndef Q_MOC_RUN // to avoid Qt4 bug, https://bugreports.qt.io/browse/QTBUG-22829
+#include <boost/math/constants/constants.hpp>
+#endif
+
+#include "MeshLib/Elements/Element.h"
+#include "MeshLib/Node.h"
+
+namespace ProcessLib
+{
+namespace SmallDeformationWithLIE
+{
+
+namespace
+{
+
+void computeNormalVector2D(MeshLib::Element const& e, Eigen::Vector3d & normal_vector)
```

Comment:
- it's only for lines.
- the normal vector is oriented in the left direction such that there is no rotation required for usual 1D line elements (I mean a result of `computeRotationMatrix2D()`)